### ISSUE TITLE
The Title Bar 

### ISSUE THREAD

**Issue by gmims** [CS-451-Spring-2015/CCSC_Submission_System#22]
We should expand the title bar on the contributor side. Users need to be able to see their full title just in case they might have a grammatical error.

**Comment by amandajcoffelt** [comment]
I think this well help make the website look more professional.
Would be nice to have separate tabs for each page that we have on the title bar verses having different links to each one.

**Comment by gmims** [comment]
I agree it would definitely make the website look more professional.
